impossibl/pgjdbc-ng
# Modernize, cleanup & fix travis config

* Use Travis “stages” to reduce redundant work
  * Checks - pre-test checks (e.g. checkstyle)
  * Test - normal test matrix
  * Deploy - snapshot artifacts & documents uploads
* Use travis conditionals for deploy stage check
  * Deploys from `develop` and `rel/.*`
* Fix filename case issues causing incorrect doc builds
* Properly fixes deploy key setup for gh-pages deploy
